### PR TITLE
Support ideaIU and WebStorm

### PR DESCRIPTION
**Pull request by vzhukovs**
### What does this PR do?
This changes proposal describes steps needed to successfully built Eclipse Che editors based on JetBrains products, especially:

- IntelliJ Idea Community Edition

- IntelliJ Idea Ultimate Edition

- WebStorm

### What issues does this PR fix or reference?
https://issues.redhat.com/browse/RHDEVDOCS-2161

### Specify the version of the product this PR applies to.
CRW 2.5

### PR Checklist

As the author of this Pull Request I made sure that:

- [ ] `vale` has been run successfully against the PR branch
- [ ] Link checker has been run successfully against the PR branch
- [ ] Documentation describes a scenario that is already covered by QE tests, otherwise an issue has been created and acknowledged by Che QE team
- [ ] Changed article references are updated where they are used (or a redirect has been set up on the docs side):
    - [ ] Dashboard [branding.json](https://github.com/eclipse/che-dashboard/blob/master/src/components/branding/branding.json)
    - [ ] Chectl [constants.ts](https://github.com/che-incubator/chectl/blob/master/src/constants.ts)

Signed-off-by: Vladyslav Zhukovskyi <vzhukovs@redhat.com>

### CODE REVIEW

**Comment by azatsarynnyy** [comment]
Hello @themr0c @MichalMaler 
Should we open another PR to 7.20.x branch in order to get in CRW 2.5 docs?

File: modules/end-user-guide/partials/con_jetbrains-ides.adoc [hunk 1]
```diff
@@ -0,0 +1,27 @@
+[id="con_jetbrains-ides_{context}"]
+= Support JetBrains IDEs
+
+In this section you'll find information about supported JetBrains IDEs
+that can be used in Eclipse Che workspace.
```
Comment (benoitf) [inline modules/end-user-guide/partials/con_jetbrains-ides.adoc]: Hello, FYI due to whitelabelling, Eclipse Che should not be used
`{prod}` should be used instead

the same for `Kubernetes` etc.

You may take a look at all attributes there: https://github.com/eclipse/che-docs/blob/master/antora-playbook.yml#L33-L112

Comment (vzhukovs) [reply modules/end-user-guide/partials/con_jetbrains-ides.adoc]: Fixed. Thanks for pointing it.

**Comment by MichalMaler** [comment]
> Hello @themr0c @MichalMaler
> Should we open another PR to 7.20.x branch in order to get in CRW 2.5 docs?

Hello Artem @azatsarynnyy , no need to create another PR. When this will be reviewed and merged, it will be also cherry-picked to 7.20.x and downstream from there to CRW 2.5 docs.

**Comment by azatsarynnyy** [comment]
Thank you @MichalMaler for answering!
Okay, waiting for @vzhukovskii to fix the issues to pass the vale-pr-check

**Comment by azatsarynnyy** [comment]
Hello
@themr0c @rkratky @MichalMaler is there anything on our side we can do to move this PR forward?

**Comment by themr0c** [comment]
> Hello @themr0c @MichalMaler
> Should we open another PR to 7.20.x branch in order to get in CRW 2.5 docs?

Usually we cherry-pick the resulting commit from master to the 7.20.x branch. We use the need-cherry-pick/7.20.x label to trigger this process.

**Comment by themr0c** [comment]
> Hello
> @themr0c @rkratky @MichalMaler is there anything on our side we can do to move this PR forward?

@rkratky has started working on it. It's a matter of human bandwidth.

**Comment by azatsarynnyy** [comment]
> > Hello
> > @themr0c @rkratky @MichalMaler is there anything on our side we can do to move this PR forward?
> 
> @rkratky has started working on it. It's a matter of human bandwidth.

Sure. Thank you @themr0c for the reply!

**Review by rkratky — CHANGES_REQUESTED** [review]
Review submitted as PR #1758.

**Comment by azatsarynnyy** [comment]
Thank you @rkratky for the review!
@vzhukovs will look at https://github.com/eclipse/che-docs/pull/1758

**Review by yhontyk — APPROVED** [review]

**Comment by azatsarynnyy** [comment]
@rkratky can this be merged? It seems everything is good now.

**Review by rkratky — APPROVED** [review]
@boczkowska Approved

**Comment by themr0c** [comment]
This PR should not have landed into 7.20.x. I am reverting the commit in that branch.

**Pull request merged**